once_test.py

```diff
@@ -519,6 +537,41 @@ def once_fn(self):
         self.assertEqual(b.once_fn(), 1)
         self.assertEqual(b.once_fn(), 1)
 
+    def test_once_per_class_parallel(self):
+        class _CallOnceClass(Counter):
+            @once.once_per_class
+            def once_fn(self):
+                return self.get_incremented()
+
+        once_obj = _CallOnceClass()
+
+        @execute_with_barrier(n_workers=_N_WORKERS)
```

Comment:
Same here, barrier should go between the once and the onced function and we can remove execute

Reply:
added a comment